release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -170,6 +170,11 @@ Previously, you could not specify the subdomain of a route, and the `spec.host`
 
 You can use this enhancement to simplify sharding by enabling a route to have multiple, distinct host names determined by each router deployment that exposes the route.
 
+[id="ocp-4-11-external-dns-operator"]
+==== External DNS Operator is GA for AWS, Azure, and GCP and is TechPreview for AWS GovCloud
+
+The External DNS Operator is GA for AWS, Azure, GCP and is TechPreview for AWS GovCloud in {product-title} {product-version}. You can now create DNS records on a public DNS zone on Infoblox by using the External DNS Operator. Previously, you needed to create multiple namespaces and roles to install the External DNS Operator. You can now use a single namespace `external-dns`. You must migrate from TechPreview to GA. For more information, see xref:../networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc#nw-installing-external-dns-operator-on-cloud-providers[About the External DNS Operator].
+
```

Comment:
This link can be added as a migration guide: https://github.com/openshift/external-dns-operator/blob/main/docs/migration-guides.md#from-01x-techpreview-to-10x-ga

Reply:
Addressed

Reply:
hmm, As per the doc guidelines we cannot link to a GitHub repo.

Reply:
I am checking how we can address this migration guide approach.